src/components/TransactionsTable/style.scss

```diff
@@ -2,63 +2,56 @@
 
 .transactions {
   .filters-row {
-    background: $gray-5;
-
     @include rem(margin-top, 61px);
   }
 
   .transaction-filters {
-    @include rem(padding, 0 76px 0 100px);
+    border-bottom: 1px solid $border-alt;
+    padding-bottom: 20px;
+
+    @include rem(margin, 0 100px);
 
     &__title {
       color: $heading;
       display: inline-block;
+      line-height: 1.35;
       font-weight: $fw-semi;
-      padding: 29px 0;
-
-      @include rem(font-size, 22px);
+      @include rem(font-size, 26px);
 
       @media all and (max-width: $screen-mobile-old) {
         display: block;
       }
     }
 
     &__item {
-      color: #636f7e;
+      color: #8c94a0;
       cursor: pointer;
       float: right;
-      font-weight: $fw-semi;
-      letter-spacing: 1.5px;
-      opacity: 0.6;
-      padding: 6px 24px;
-      text-transform: uppercase;
-      margin: 24px 0 0 6px;
+      margin: 6px 0 0 25px;
+      padding-bottom: 26px;
 
-      @include rem(font-size, 15px);
-      @include rem(padding-left, 24px);
-      @include rem(padding-right, 24px);
+      @include rem(font-size, 18px);
 
       &_active {
-        background: #0388ef;
-        border-radius: 100px;
-        color: $white;
-        opacity: 1;
-      }
-
-      @media all and (max-width: $screen-mobile-old) {
-        margin: 0 0 29px 6px;
+        border-bottom: 2px solid $blue;
+        border-radius: 1px;
+        color: $heading;
+        font-weight: $fw-semi;
+        margin-top: 9px;
       }
     }
-
-    @media all and (max-width: $screen-tablet) {
-      @include rem(padding, 0);
-    }
   }
 
   .transactions-table {
     @include rem(margin-top, 12px);
     @include rem(padding, 0 94px);
 
+    &__asset-col,
```

Comment:
Just `__column`

Reply:
`column` isn't element of block `asset`, `asset column` is element of block `transactions-table`, also element `asset` can't contain another element https://en.bem.info/methodology/quick-start/#element